presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcSelectivePageSourceFactory.java

```diff
@@ -398,6 +399,10 @@ public static OrcSelectivePageSource createOrcPageSource(
             }
             catch (IOException ignored) {
             }
+            // instanceof check does not work here since e and UncheckedExecutionException are loaded by different classloaders
```

Comment:
also talk about why .class comparison does work either, can be something like
"instanceof and class comparison do not work as they are loaded by different class loaders."